src/main/java/com/sonyericsson/jenkins/plugins/bfa/PluginImpl.java

```diff
@@ -437,6 +462,20 @@ public void setNrOfScanThreads(int nrOfScanThreads) {
     }
 
     /**
+     * Checks if the build with certain result should be analyzed or not.
+     *
+     * @param result the result
+     * @return true if it should be analyzed.
+     */
+    public static boolean needToAnalyze(Result result)  {
+        if (getInstance().doNotAnalyzeAbortedJob()) {
```

Comment:
Thanks you. It works, but I decide to avoid this check. But I added information how to handle case with aborted jobs to help.
Main point why I don't want to add this check - because I wish to have black list (all aborted jobs except time-outed) instead of white (only aborted by user). But I can't implement it, because I don't want to add dependency for "timeout" plugin.